### PR TITLE
docs(tutorial): improved <Slot /> description

### PR DESCRIPTION
**Pull request by KenAKAFrosty**
As per the brief exchange over in discord:
https://discord.com/channels/842438759945601056/996551595359740145/1022017984245530664

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

Moved content into parent so `<Slot />` usage is purely when/where

# Use cases and why

Based on the overview of the philosophy behind `<Slot />` provided in projection/basic, this seemed more true to the spirit of <Slot /> preferred usage. @mhevery liked the idea when mentioned in Discord and suggested providing the update. Happy to help! 🙂

# Checklist:

- [X] My code follows the [developer guidelines of this project](../CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/tutorial/projection/slots/index.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ In simple cases, projection allows content from the parent component to be proje
 
 ### Example
 
-In this example, we have created `<Collapsable>` component that toggles between open and closed states. Currently, when the `<Collapsable>` is closed it does not show any content. Add additional `<Slot>` to project the `q:slot="closed"` content.
+In this example, we have created `<Collapsable>` component that toggles between open and closed states. Currently, when the `<Collapsable>` is closed it shows its own content. Change that to an additional `<Slot>` to project the `q:slot="closed"` content from the parent instead.
```
Comment (shairez) [inline packages/docs/src/routes/tutorial/projection/slots/index.mdx:9]: Thanks!
I'm not sure I get "it shows its own content"

Can you clarify please?

**Comment by shairez** [comment]
Thanks a lot @KenAKAFrosty !

Can you please fix the linting / prettier errors locally and push?

Plus, I've added one comment

Thanks!

**Comment by KenAKAFrosty** [comment]
> Thanks a lot @KenAKAFrosty !
> 
> Can you please fix the linting / prettier errors locally and push?
> 
> Plus, I've added one comment
> 
> Thanks!

Okay, got that all cleaned up! I reworded the index.mdx file as well - does that make more sense? Would love your input! Thanks

**Comment by shairez** [comment]
Great job @KenAKAFrosty 
Thanks!

**Pull request merged**